src/OrbitBase/File.cpp

```diff
@@ -92,6 +92,15 @@ ErrorMessageOr<unique_fd> OpenFileForWriting(const std::filesystem::path& path)
   return OpenFile(path, kOpenFlags, kOpenMode);
 }
 
+ErrorMessageOr<unique_fd> OpenExistingFileForReadWrite(const std::filesystem::path& path) {
```

Comment:
I would say to move it after `OpenNewFileForReadWrite` to keep the same order as in the `.h`.

Reply:
done.